.ci/jenkins/Jenkinsfile.e2e.cluster

```diff
@@ -132,13 +167,18 @@ void setupCluster() {
     }
 }
 
-void setupMinikube() {
-    // Start minikube
-    minikube.minikubeMemory = '12g'
-    minikube.start()
+void installKind() {
+    return sh(script: "command -v kind >/dev/null || go install sigs.k8s.io/kind@${KIND_VERSION}", returnStdout: true).trim()
```

Comment:
Can this shell be moved to kind.groovy file instead?

Reply:
Sure, makes sense